src/main/scala/com/gu/fastly/Lambda.scala

```diff
@@ -104,13 +135,6 @@ class Lambda {
     dotcomSurrogateKey
   }
 
-  private def sendFastlyPurgeRequestAndAmpPingRequest(contentId: String, purgeType: PurgeType, serviceId: String, surrogateKey: String, fastlyApiKey: String): Boolean = {
-    if (sendFastlyPurgeRequest(contentId, purgeType, serviceId, surrogateKey, fastlyApiKey))
-      AmpFlusher.sendAmpDeleteRequest(contentId)
```

Comment:
Has the AmpFlusher.sendAmpDeleteRequest been omitted in this branch?

Reply:
Yes I couldn't see anything using it (and it's marked private so shouldn't have to look far)